config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'users/new'
```

Comment:
Recommend use resources
```suggestion
  resources :users
```